README.md

```diff
@@ -43,6 +47,54 @@ Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
+## Styles
+
+The project uses Material-UI's styling solution for components (see the [documentation](https://material-ui.com/styles/basics/)).
+
+For example, a functional component can be styled using hooks:
+
+```tsx
+import React from 'react';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  root: { 'font-weight': 'bold' },
+});
+
+const Example: React.FC = () => {
+  const classes = useStyles();
+  return <div className={classes.root}>Hello world!</div>;
+};
+
+export default Example;
+```
+
+## i18n
+
+Locales are defined in [src/locales/en.json](./src/locales/en.json) like this:
```

Comment:
If possible it would be awesome to have a script that automtically extracts all translation keys

Reply:
This is possible and there are some [tools](https://www.i18next.com/overview/plugins-and-utils#extraction-tools) available, but I'd prefer keeping it out of this PR.